requirements/examples.txt

```diff
@@ -1,2 +1,3 @@
 torchvision>=0.5
 gym>=0.17.0
+ipython[all]
```

Comment:
It seems to be getting installed in the Docker image as well?

Reply:
@kaushikb11 Yeah, this was a hack to get it to work with PL 1.2.7 - the docker image grabs old requirements and so doesn't necessarily install `ipython[all]` (which is needed for it to work), so has to be manually installed in the `Dockerfile` too.

Reply:
anyway we have it docker as if-needed so no reinstall :]